qiskit/tools/wigner_function_visualisation.py

```diff
@@ -0,0 +1,264 @@
+    ax.set_yticks([], [])
+    ax.set_zticks([], [])
+    ax.grid(False)
+    ax.xaxis.pane.set_edgecolor('black')
+    ax.yaxis.pane.set_edgecolor('black')
+    ax.zaxis.pane.set_edgecolor('black')
+    ax.set_xlim(-1.5, 1.5)
+    ax.set_ylim(-1.5, 1.5)
+    ax.set_zlim(-1.5, 1.5)
+    m = cm.ScalarMappable(cmap=cm.seismic_r)
+    m.set_array([-Wmax, Wmax])
+    plt.colorbar(m, shrink=0.5, aspect=10)
+    
+    plt.show()
+
+
+def plot_wigner_curve(wigner_results, xaxis=None):
+    """
+    Plots a curve for points in phase space of the spin Wigner function
+    
+    Args:
+        wigner_results(np.array): an array of points to plot as a 2d curve
+        xaxis (np.array):  the range of the x axis
+
+    Returns:
+        none: plot is shown with matplotlib to the screen
+    
+    """
+    if xaxis is None:
+        xaxis = np.linspace(0,len(wigner_results)-1,num=len(wigner_results))
+    
+    plt.plot(xaxis,wigner_results)
+    plt.show()
+
+
+def plot_wigner_plaquette(wigner_results, maxWigner='local'):
+    """
+    Plots plaquette of wigner function data, the plaquette will consist
+    of cicles each colored to match the value of the Wigner function at
+    the given point in phase space.
+
+    Args:
+        wigner_results (matrix): array of Wigner function data where the
+                                 rows are plotted along the x axis and the
+                                 columns are plotted along the y axis
+        maxWigner: - 'local' puts the maximum value to maximum of the points
+                   - 'unit' sets maximum to 1
+                   - float for a custom maximum.
+
+    Returns:
+        none: plot is shown with matplotlib to the screen
+
+    """
+    wigner_results = np.matrix(wigner_results)
+    dim = wigner_results.shape
+    
+    if maxWigner == 'local':
+        Wmax = np.amax(wigner_results)
+    elif maxWigner == 'unit':
+        Wmax = 1
+    else:
+        Wmax = maxWigner #For a float input
+        
+    cmap = matplotlib.cm.get_cmap('seismic_r')
+    
+    xax = dim[1]-0.5
+    yax = dim[0]-0.5
+    norm = np.amax(dim)
+    
+    fig = plt.figure(figsize=((xax+0.5)*6/norm,(yax+0.5)*6/norm))
+    ax = fig.gca()
+
+    for x in range(int(dim[1])):
+        for y in range(int(dim[0])):
+            
+            circle = plt.Circle((x,y),0.49,
+                                color =
+                                cmap((wigner_results[y,x]+Wmax)/(2*Wmax)))
+            ax.add_artist(circle)
+
+    ax.set_xlim(-1, xax+0.5)
+    ax.set_ylim(-1, yax+0.5)
+    ax.set_xticks([], [])
+    ax.set_yticks([], [])
+    m = cm.ScalarMappable(cmap=cm.seismic_r)
+    m.set_array([-Wmax, Wmax])
+    plt.colorbar(m, shrink=0.5, aspect=10)
+    plt.show()
+
+
+def plot_wigner_function(wigner_results, phis=None,
+                         thetas=None, method=None,
+                         text_out=None):
+    """
+    Plots Wigner results in appropriate format
+
+    Args:
+        wigner_results:
```

Comment:
I'd change wigner_results to `wigner_data` and mention in the description that this is the output returned from the wigner_data` function.